backend/src/v5/middleware/auth.js

```diff
@@ -24,7 +25,25 @@ const { validateMany } = require('./common');
 
 const AuthMiddleware = {};
 
-AuthMiddleware.validSession = async (req, res, next) => {
+const isIpAddressValid = async (req, res, next) => {
```

Comment:
function name is a bit misleading, should be something like `ipAddressMatched`?

Reply:
we also should be matching user agent (i.e. if someone logged in via chrome but now trying to use the same cookie on firefox that should not work)

Reply:
we are currently not storing the browser in the session, just whether it is a web session or not

![image](https://github.com/user-attachments/assets/dd2e94ea-8300-4e46-b0c3-41ffe0ffd3e3)

should I change it so we store the browser upon login as well ?